keystore-service/src/main/java/com/stnetix/ariaddna/keystore/utils/KeyFactory.java

```diff
@@ -0,0 +1,153 @@
+package com.stnetix.ariaddna.keystore.utils;
+
+
+import com.stnetix.ariaddna.commonutils.logger.AriaddnaLogger;
+import com.stnetix.ariaddna.keystore.exceptions.KeyStoreException;
+import sun.security.x509.X509CertImpl;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.security.KeyStore;
+
+/**
+ * Created by alexkotov on 20.04.17.
+ */
+public class KeyFactory {
+    private static final KeyFactory KEY_FACTORY = new KeyFactory();
+    private KeyFactory(){}
+
+    public static KeyFactory getKeyFactory() {
+        return KEY_FACTORY;
+    }
+
+    private static final String KEYSTORE_PATH;
+    private static final String DISABLED_KEYSTORE_PATH;
+    static final char[] PASS;
+    private static final AriaddnaLogger LOGGER;
+
+    static {
+        PASS = "password1".toCharArray();
+        KEYSTORE_PATH = "ariaddna.keystore";
+        DISABLED_KEYSTORE_PATH = "disabled_ariaddna.keystore";
```

Comment:
We can't mark certificate as disable. For this purpose i intodused disabled keystore.